test/config-luatest/mkdir_test.lua

```diff
@@ -0,0 +1,24 @@
+local t = require('luatest')
+local helpers = require('test.config-luatest.helpers')
+
+local g = helpers.group()
+
+g.test_startup_success = function(g)
```

Comment:
The test case name is accidentally copy-pasted from the `app_test.lua`. Let's adjust.

Reply:
Renamed to `test_nested_dirs`.